entrypoint.sh

```diff
@@ -32,7 +32,8 @@ jq_run "git config --global user.name '$GIT_USER'"
 jq_run "dvc init" noexit
 jq_run "dvc config core.autostage true"
 jq_run "dvc config core.analytics false"
-wurzel generate $WURZEL_PIPELINE --data-dir $DVC_DATA_PATH >> $DVC_FILE || exit 1
+ . ${VENV}/bin/activate && wurzel generate $WURZEL_PIPELINE --data-dir $DVC_DATA_PATH > $DVC_FILE || exit 1
```

Comment:
```suggestion
wurzel generate $WURZEL_PIPELINE --data-dir $DVC_DATA_PATH > $DVC_FILE || exit 1
```
Is the venv activation necessary, since it is already done in the Dockerfile ?